docs/api/commands/press.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: 'cy.press() | Cypress Documentation'
+description: Trigger native key events in your application to simulate keyboard interactions.
+sidebar_label: press
+slug: /api/commands/press
+componentSpecific: false
+---
+
+<ProductHeading product="app" />
+
+# press
+
+Trigger native key events in your application to simulate keyboard interactions.
+
+A `keydown`, `press`, and `keyup` event will be dispatched directly to the browser window. Note that your application must have focus to properly receive this event. It can be helpful to use `cy.focus()` to ensure focus on an element in your application before using this command.
+
+Unlike `cy.type()`, which is best for typing character keys, `cy.press()` will dispatch real keyboard events rather than simulated ones. This command is especially useful when testing focus management and keyboard navigation patterns which are critical for [accessibility testing](/app/guides/accessibility-testing) and great keyboard UX.
+
+Currently, the only key supported is `Tab`.
+
+:::caution
+
+<strong>Supported Browsers:</strong>
+This command is supported in chromium browsers and Firefox versions >=
+v135. WebKit is not supported. This command will fail if executed in a browser that
+does not support it.
+
+:::
+
+## Syntax
+
+```javascript
+cy.press(key)
+cy.press(key, options)
+```
+
+## Signature
+
+```typescript
+interface PressCommand {
+  (
+    key: KeyPressSupportedKeys,
+    options?: Partial<Cypress.Loggable> & Partial<Cypress.Timeoutable>
+  ): void
+}
+```
+
+## Usage
+
+<Icon name="check-circle" color="green" /> **Correct Usage**
+
+```javascript
+cy.get('input.first').focus()
+cy.press(Cypress.Keyboard.Keys.TAB)
+cy.get('input.second').should('have.focus')
+```
+
+<Icon name="exclamation-triangle" color="red" /> **Incorrect Usage**
+
+```javascript
+cy.get('input.first').focus()
+cy.press(Cypress.Keyboard.Keys.TAB)
+  // Errors because press yields null
+  .should('have.focus') 
+```
+
+### Arguments
+
+<Icon name="angle-right" /> **key _(String)_**
+
+The key to press. The supported values are available on `Cypress.Keyboard.Keys`, and may change from time to time. It's recomended that you reference these values from `Cypress.Keyboard.Keys` rather than passing in a string.
```

Comment:
are we able to link out to `Cypress.Keyboard.Keys` to make the values clearer to the user?

Reply:
right now it's just "TAB" though, right?